velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -719,6 +894,14 @@ static bool compareMaterializedRows(
     }
   }
 
+  // In case there are values in right but not in left, check the other way
```

Comment:
Perhaps, clarify that this applies to cases where there are duplicate values in 'left'. If all values in 'left' are unique, this additional loop is not necessary.